lagotto/couchdb.sls

```diff
@@ -2,12 +2,16 @@
 #
 # CouchDB only supports binding to single or all interfaces (ie, not selective)
 # https://issues.apache.org/jira/browse/COUCHDB-907
```

Comment:
This comment is a CouchDB 1.x issue and has been closed (as won't fix). I hesitate to delete because comment was added by original author. Maybe just update comment to refer to 1.x (CouchDB 1.x ...). This may not be an issue for 2.x.

Reply:
This looks to still be an issue, https://github.com/apache/couchdb/issues/1589 I'll update the comment to point to the new issue.